### ISSUE TITLE
Feature request, hide module(and it's depencencies)

### ISSUE THREAD

**Issue by tiye** [webpack/analyse#11]
I was trying to analyse my project dependency for reusing code in another project which is nearly the same. But found the are over 500 modules, with only 230 of them are belonging to my project. So I think if I can hide modules just by filtering or shortcut, that would be much more helpful.

By now my only solution is for read `asset.json` as object and filter them by hand and save it again.

**Comment by acthp** [comment]
Agreed. Hiding deps (accounting size to the parent) and sorting by size are critical. Drawing a hairball is not useful. Listing hundreds of deps un-ordered is not useful.

**Comment by bendavis78** [comment]
+1

**Comment by a-r-d** [comment]
@jiyinyiyong this sounds like a good feature, make a PR if you can. 

@acthp I just merged this PR which adds sorting: https://github.com/webpack/analyse/pull/16

**Comment by alerkesi** [comment]
+1

**Comment by evenstensberg** [comment]
This is implemented in master now, you can have a look yourselves: https://webpack.github.io/analyse/ 

Thanks for the issue, a bit late sorry about that! 

<img width="1439" height="695" alt="Image" src="https://github.com/user-attachments/assets/b755ea12-97dd-474c-818b-430b0231bee1" />
